build/fix-api-docs.js

```diff
@@ -5,7 +5,26 @@ var apiPath = path.join(__dirname, '..', 'docs', 'api');
 var replacements = [
   {find: /\/docs\/guides\/(.+)\.md/g, replace: 'tutorial-$1.html'},
   {find: /tutorial-tech.html/g, replace: 'tutorial-tech_.html'},
-  {find: /\/docs\/guides\//g, replace: '#'}
+  {find: /\/docs\/guides\//g, replace: '#'},
+  {find: /(\<h[1-6] id="(?:.*)?)video-js(.*)?"\>/g, replace: '$1videojs$2">'},
+  {find: /(\<h[1-6] id="(?:.*)?)don-t(.*)?"\>/g, replace: '$1dont$2">'},
+  {find: /(\<h[1-6] id="(?:.*)?)node-js(.*)?"\>/g, replace: '$1nodejs$2">'},
+  {find: /(\<h[1-6] id="(?:.*)?)vtt-js(.*)?"\>/g, replace: '$1vttjs$2">'},
+  {find: /(\<h[1-6] id=")-(.*)("\>)/g, replace: '$1$2$3'},
+  {find: /(\<h[1-6] id=")(.*)-("\>)/g, replace: '$1$2$3'},
+  {find: /(\<h[1-6] id=".*)-docs-guides-.*-md("\>)/g, replace: '$1$2'},
+  {find: '<h3 id="videojs-(audio|video)track">', replace: '<h3 id="videojs$1track">'},
+  {find: '<h3 id="text-tracks">', replace: '<h3 id="text-tracks-1">'},
+  {find: '<h2 id="q-how-can-i-hide-the-links-to-my-video-subtitles-audio-tracks">',
+   replace: '<h2 id="q-how-can-i-hide-the-links-to-my-videosubtitlesaudiotracks">'},
+  {find: '<h3 id="dispose-http-docs-videojs-com-player-html-dispose">',
+   replace: '<h3 id="dispose">'},
+  {find: '<h4 id="effect-on-player-width-and-player-height">',
+   replace: '<h4 id="effect-on-playerwidth-and-playerheight">'},
+  {find: '<h4 id="i-want-to-have-a-single-source-and-dont-care-about-live-adaptive-streaming">',
+   replace: '<h4 id="i-want-to-have-a-single-source-and-dont-care-about-liveadaptive-streaming">'},
+  {find: '<h2 id="api-docs-api">', replace: '<h2 id="api-docs">'},
+  {find: '<h2 id="guides-docs-guides">', replace: '<h2 id="guides">'}
```

Comment:
I guess my only question with this section is whether it would make more sense to parse the HTML into a DOM and loop through all the heads, looking at their `id` attributes? It might be less error/typo-prone than this approach.